### DISCUSSION TITLE
setUserPerm

### DISCUSSION THREAD

**Discussion by seohayeon** [storycraft/node-kakao#704] [Q&A] [answered]
setUserPerm을 쓰려고 하는데 -203 오류가 납니다.
제 코드는 다음과 같습니다.
`await channel.setUserPerm({ userId: new Long("363097969") },8)` 

result
`{ status: -203, success: false }`

오픈채팅방에 일반 유저를 봇으로 바꾸고 싶습니다

**Comment by benesiaxd** [comment, marked as answer]
Long 말고 Number 로 넘겨도안되나요?
묵시적으로 Number로 바뀌긴할텐데 안되면 스트링도 해보시고 안되면 막힌겁니다

**Reply by seohayeon** [reply]
일반유저랑 부방장은 잘되는데 8(봇) 권한만 안됩니다

**Reply by benesiaxd** [reply]
그럼더더욱 막힌거겠네용
